modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestinationOptions.java

```diff
@@ -32,6 +32,10 @@
 public class HTTPDestinationOptions {
 
   public static String URL = "url";
+  public static String METHOD  = "method";
+  public static String METHOD_DEFAULT  = "PUT";
+
+  public static HashSet<String> HTTP_METHODS = new HashSet<String>(Arrays.asList("POST", "PUT"));
```

Comment:
totally, I was just giving examples: the method should either be free text or be an enum list of all possible values in the REST spec